src/main/java/de/tum/in/www1/artemis/service/PrivacyStatementService.java

```diff
@@ -0,0 +1,129 @@
+
+    private static final String PRIVACY_STATEMENT_FILE_EXTENSION = ".md";
+
+    /**
+     * Returns the privacy statement if you want to update it.
+     * If it currently doesn't exist an empty string is returned as it will be created once the user saves it.
+     *
+     * @param language the language of the privacy statement
+     * @return the privacy statement with the given language
+     */
+    public PrivacyStatement getPrivacyStatementForUpdate(PrivacyStatementLanguage language) {
+        String privacyStatementText = "";
+        if (getPrivacyStatementPath(language).isEmpty()) {
+            return new PrivacyStatement(privacyStatementText, language);
+        }
+        try {
+            privacyStatementText = Files.readString(getPrivacyStatementPath(language).get());
+        }
+        catch (IOException e) {
+            log.error("Could not read privacy statement file for language {}", language);
+            throw new InternalServerErrorException("Could not read privacy statement file for language " + language);
+        }
+        return new
+
+        PrivacyStatement(privacyStatementText, language);
+
+    }
+
+    /**
+     * Returns the privacy statement if you want to view it
+     * If it currently doesn't exist in the given language, the other language is returned.
+     * If it currently doesn't exist in any language, an exception is thrown.
+     *
+     * @param language the language of the privacy statement
+     * @return the privacy statement with the given language
+     */
+
+    public PrivacyStatement getPrivacyStatement(PrivacyStatementLanguage language) {
+        String privacyStatementText;
+        // if it doesn't exist for one language, try to return the other language, and only throw an exception if it doesn't exist for both languages
+        if (getPrivacyStatementPath(PrivacyStatementLanguage.GERMAN).isEmpty() && getPrivacyStatementPath(PrivacyStatementLanguage.ENGLISH).isEmpty()) {
+            throw new BadRequestAlertException("Could not find privacy statement file for any language", "privacyStatement", "noPrivacyStatementFile");
+        }
+        else if (language == PrivacyStatementLanguage.GERMAN && getPrivacyStatementPath(language).isEmpty()) {
+            language = PrivacyStatementLanguage.ENGLISH;
+        }
+        else if (language == PrivacyStatementLanguage.ENGLISH && getPrivacyStatementPath(language).isEmpty()) {
+            language = PrivacyStatementLanguage.GERMAN;
+        }
+
+        try {
+            privacyStatementText = Files.readString(getPrivacyStatementPath(language).get());
+        }
+        catch (IOException e) {
+            log.error("Could not read privacy statement file for language {}", language);
+            throw new InternalServerErrorException("Could not read privacy statement file for language " + language);
+        }
+        return new PrivacyStatement(privacyStatementText, language);
```

Comment:
```suggestion
        try {
            String privacyStatementText = Files.readString(getPrivacyStatementPath(language).get());
            return new PrivacyStatement(privacyStatementText, language);
        }
        catch (IOException e) {
            log.error("Could not read privacy statement file for language {}", language);
            throw new InternalServerErrorException("Could not read privacy statement file for language " + language);
        }
```
Also here: returning directly from the happy-path instead of jumping out of the try and then returning at the end would be more intuitive for me (maybe personal).